phpBB/styles/prosilver/template/smiley_dropdown.html

```diff
@@ -0,0 +1,41 @@
+<div id="smiley-dropdown" class="dropdown dropdown-extended" >
+    <div class="pointer"><div class="pointer-inner"></div></div>
+    <div class="smiley-content">
+        <div id="smiley-box" class="smiley-box">
+            <ul>
+            <!-- EVENT posting_editor_smilies_before -->
+            <!-- IF S_SMILIES_ALLOWED and .smiley -->
+
+            <!-- BEGIN smiley -->
+                <li class="smiley-list">
+            <a href="#" onclick="insert_text('{smiley.A_SMILEY_CODE}', true); return false;"><img src="{smiley.SMILEY_IMG}" width="{smiley.SMILEY_WIDTH}" height="{smiley.SMILEY_HEIGHT}" alt="{smiley.SMILEY_CODE}" title="{smiley.SMILEY_DESC}" /></a>
```

Comment:
js code should never be inserted into your html use jquery and data elements to abstract it into the forum_fn.js file